src/sdk/bucketspace_fs.js

```diff
@@ -87,10 +107,20 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
         }
     }
 
+    _translate_bucket_error_codes(err) {
+        if (err.rpc_code) return err;
+        if (err.code === 'ENOENT') err.rpc_code = 'NO_SUCH_BUCKET';
+        if (err.code === 'EEXIST') err.rpc_code = 'BUCKET_ALREADY_EXISTS';
+        if (err.code === 'EPERM' || err.code === 'EACCES') err.rpc_code = 'UNAUTHORIZED';
+        if (err.code === 'IO_STREAM_ITEM_TIMEOUT') err.rpc_code = 'IO_STREAM_ITEM_TIMEOUT';
+        if (err.code === 'INTERNAL_ERROR') err.rpc_code = 'INTERNAL_ERROR';
+        return err;
+    }
```

Comment:
also here - this function was deleted in PR #8224

Reply:
Probably a bad merge.
I will remove.